### ISSUE TITLE
VoiceOver "Back from email is getting into loop

### ISSUE THREAD

**Issue by BijinDev** [tutao/tutanota#8398]
While using VoiceOver, open an email and close that email by clicking back button leads to loop by saying "back" when we move forward or backward

**Steps to reproduce**
-   Turn on VoiceOver in iphone and open an email
-   Close that email by clicking back(<) button and get back to list view.
-   Then try to move forward and backward without selecting anything. It will keep on saying "back"

**Expected behavior**

-  When closing an email and coming back to mail list, should be able to move forward and backward to other options by swiping left or right. 
 
**Estimated: 10h**
